symengine/sets.h

```diff
@@ -249,6 +249,39 @@ class Complement : public Set
     }
 };
 
+class ConditionSet : public Set
+{
+private:
+    vec_sym syms_;
```

Comment:
https://github.com/sympy/sympy/blob/master/sympy/solvers/tests/test_solveset.py#L1383
This is the only point I could find where SymPy actually uses multiple symbols for ConditionSet.
Should we make it only for a single variable then?